__tests__/app.test.js

```diff
@@ -129,4 +129,103 @@ describe("/api/articles/:article_id", () => {
         });
     });
   });
+  describe("PATCH", () => {
+    test("Status:201, the body should contain a property of inc_votes and should have a number as a value", () => {
+      const addVotes = { inc_votes: -100 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(addVotes)
+        .expect(201)
+        .then(({ body }) => {
+          expect(addVotes).toHaveProperty("inc_votes");
+          expect(addVotes.inc_votes).toEqual(expect.any(Number));
+        });
+    });
+    test("Status:201, should update votes for the requested article_id", () => {
+      const addVotes = { inc_votes: 100 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(addVotes)
+        .expect(201)
+        .then(({ body: { articles } }) => {
+          articles.forEach((article) => {
```

Comment:
Why is there a forEach when the article is a single object? Depending on how you've sent the data back, this should be on a key of article as it is singular. In that case all you need is line 153.